src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -266,3 +267,20 @@ export const convertProgramDataOptionToRangeOption = (
       );
   }
 };
+
+export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElement => {
+  const startIndex = conceptName.toLowerCase().indexOf(searchFilter.toLowerCase());
```

Comment:
I wonder if there's a way to use `.split()` here to make the match do the substring for us? Something like
```
  const splits = conceptName.split(new RegExp(searchFilter, 'i'));
  if (splits.length !== 1) {
   ....
```

I guess the problem there is that you'd lose the casing of the split area, also it would match all elements not just the first.